### ISSUE TITLE
How to cancel uploading files to S3?

### ISSUE THREAD

**Issue by giray123** [rowanwins/vue-dropzone#514]
I would like to cancel uploading files programmatically with a user event (clicking a stop button while uploading). How can I accomplish it?

**Comment by yabuking84** [comment]
Yes, how do we cancel uploading files?? Also how do we cancel uploading files to AWS?

**Comment by rowanwins** [comment]
Probably something like
````
const files = this.$refs.myVueDropzone.getUploadingFiles()
files[0].xhr.abort()
````

FOr AWS its probably trickier because our implementation is not very sophistaicated.

**Comment by rowanwins** [comment]
Underway in #549
